### DISCUSSION TITLE
Codespaces questions

### DISCUSSION THREAD

**Discussion by Romamy233** [community/community#55251] [Codespaces]
Hello.I'm new to Codespaces.I have applied to Github Students packs so I 'm a Githubpro now.But  I don't know how to build a 16GB RAM codespace,can someone tell me how to make it?thanks

**Comment by hrishikesh-kocharekar-dev** [comment]
Hello! To create a Codespace with 16GB RAM, you can follow these steps:

Go to your Github account and navigate to your repository where you want to create the Codespace.
Click on the "Code" button and select "Open with Codespaces" from the dropdown menu.
In the Codespaces menu, click on the "New codespace" button.
In the "Create a codespace" form, select the "Custom" option under "Environment".
Under the "Environment" section, select "Ubuntu 20.04" as the operating system.
Under the "Instance type" section, select "Custom".
Set the "CPU cores" to your desired number (e.g. 2).
Set the "Memory" to 16GB.
Click on the "Create codespace" button to create your new Codespace.
Your new Codespace should now have 16GB of RAM. Keep in mind that custom instances may have additional costs associated with them, so be sure to check the pricing information before creating a custom instance.
